.npmrc

```diff
@@ -0,0 +1 @@
+@cahllagerfeld:registry=https://npm.pkg.github.com
```

Comment:
i'm not sure if it's a good idea, to commit the `.npmrc` - Theres no token inside it but I'm still not sure.

Reply:
I was thinking the same, I am not sure, I think it is ok, because there is nothing private and will not effect any other project

Reply:
From GitHub docs...

> You also need to add the .npmrc file to your project so all requests to install packages will go through GitHub Packages